lib/Sema/MiscDiagnostics.h

```diff
@@ -51,8 +51,11 @@ void performTopLevelDeclDiagnostics(TypeChecker &TC, TopLevelCodeDecl *TLCD);
 /// Emit a fix-it to set the access of \p VD to \p desiredAccess.
 ///
 /// This actually updates \p VD as well.
-void fixItAccess(InFlightDiagnostic &diag, ValueDecl *VD,
-                 AccessLevel desiredAccess, bool isForSetter = false);
+void fixItAccess(InFlightDiagnostic &diag,
+                 ValueDecl *VD,
+                 AccessLevel desiredAccess,
+                 bool isForSetter = false,
+                 bool shouldNotReplace = false);
```

Comment:
Nitpick: "shouldNotReplace" doesn't tell us what to do instead. Maybe "shouldUseDefaultAccess" or something?

Reply:
Yeah, `shouldUseDefaultAccess ` sounds better!